python/src/scipp/neutron/instrument_view.py

```diff
@@ -90,11 +90,8 @@ def __init__(self, scipp_obj=None, bins=None, masks=None, cmap=None,
                                "are a Dataset or a DataArray (and their "
                                "respective proxies).".format(tp))
 
-        # Get detector positions from either coordinates or labels
-        if sc.Dim.Position in scipp_obj.coords:
-            self.det_pos = np.array(scipp_obj.coords[sc.Dim.Position].values)
-        else:
-            self.det_pos = np.array(scipp_obj.labels["position"].values)
+        # Get detector positions
+        self.det_pos = np.array(sn.position(scipp_obj).values)
```

Comment:
Is the `np.array` required?

Reply:
yes, it seems so, if I then want to slice it using `det_pos[:, 0]` to get all x coordinates.
If not, it returns an array of tuples each holding 3 numbers.

Reply:
:+1: